### DISCUSSION TITLE
[Feature Request] Separate Tap Addition texture support for noteskins

### DISCUSSION THREAD

**Discussion by chairodactyl** [TeamRizu/OutFox#231] [03 - Feature Requests]
Please add support for a separate noteskin texture for Tap Additions (in the same manner as Tap Mine, Tap Lift, etc). Currently the game only supports Addition textures as frames in the Tap Note texture and uses a texture offset in the metrics which is extremely unintuitive.
Thanks!

**Comment by chairodactyl** [comment]
Looking closer at existing noteskins (global/FourV2 specifically) I see that the metrics provide for Additions for each note type. Could support be provided to have separate textures for each type of tap addition? (Tap Mine Addition, Tap Hold Head Active Addition, etc)?

**Comment by Scraticus** [comment]
We'll look at returning this functionality, moving it to a discussion so it's easier to track
